[2024-12-14T11:20:56.392+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-14T11:20:56.633+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_program_course manual__2024-12-14T11:20:47.303390+00:00 [queued]>
[2024-12-14T11:20:56.647+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.insert_data.insert_data_program_course manual__2024-12-14T11:20:47.303390+00:00 [queued]>
[2024-12-14T11:20:56.648+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-14T11:20:56.666+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): insert_data.insert_data_program_course> on 2024-12-14 11:20:47.303390+00:00
[2024-12-14T11:20:56.675+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=558) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-14T11:20:56.677+0000] {standard_task_runner.py:63} INFO - Started process 560 to run task
[2024-12-14T11:20:56.683+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'insert_data.insert_data_program_course', 'manual__2024-12-14T11:20:47.303390+00:00', '--job-id', '116', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmpyvbtg65d']
[2024-12-14T11:20:56.688+0000] {standard_task_runner.py:91} INFO - Job 116: Subtask insert_data.insert_data_program_course
[2024-12-14T11:20:56.762+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_data.insert_data.insert_data_program_course manual__2024-12-14T11:20:47.303390+00:00 [running]> on host 472b63b715e1
[2024-12-14T11:20:56.887+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='insert_data.insert_data_program_course' AIRFLOW_CTX_EXECUTION_DATE='2024-12-14T11:20:47.303390+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-14T11:20:47.303390+00:00'
[2024-12-14T11:20:56.889+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-14T11:20:57.229+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-12-14T11:20:57.232+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-14T11:20:57.252+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=insert_data.insert_data_program_course, run_id=manual__2024-12-14T11:20:47.303390+00:00, execution_date=20241214T112047, start_date=20241214T112056, end_date=20241214T112057
[2024-12-14T11:20:57.296+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-14T11:20:57.364+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-14T11:20:57.365+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
